pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Monitor TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Monitor TCP sockets from Loopback interfaces:
+
+`nettop -m {{tcp}} -t {{loopback}}`
+
+- Monitor a specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Display summary only for per-process:
```

Comment:
```suggestion
- Display a summary only for per-process:
```

Reply:
Another suggestion:
```suggestion
- Display a per-process summary only:
```

Reply:
That sounds weird

Reply:
I think `Display a summary for per-process` is good 😆

Maybe we don't need word _only_, I copy the word _only_ from the man page:

```plaintext
     -P       Display per-process summary only, skipping details of open con-
	      nections.  This is equivalent to selecting "Collapse All" in the
	      interactive menu.
```

Reply:
I think `per-process` is used as an adjective here, so `a per-process summary` would be the correct way. Maybe @sbrl could help out here.